lib/Serialization/SerializedModuleLoader.cpp

```diff
@@ -596,17 +596,56 @@ std::string SerializedModuleBaseName::getName(file_types::ID fileTy) const {
   return std::string(result.str());
 }
 
+llvm::Optional<std::string> SerializedModuleLoaderBase::getPackageInterfacePathIfInSamePackage(llvm::vfs::FileSystem &fs, ASTContext &ctx) {
```

Comment:
Already chatted with Ellie offline: at this point, it's better to put reading the package interface behind an env variable. After we ensure package interfaces are all healthy, we could lift the environment variable guard.

Reply:
Sg. Will add an env var as well as a flag.